lib/controllers/ServiceFabrikApiController.js

```diff
@@ -474,6 +475,63 @@ class ServiceFabrikApiController extends BaseController {
         .send({}));
   }
 
+  scheduleUpdate(req, res) {
+    req.manager.isAutoUpdatePossible();
+    if (_.isEmpty(req.body.repeatInterval)) {
+      throw new BadRequest('repeatInterval is mandatory');
+    }
+    return Promise
+      .try(() => {
+        if (req.entity.name) {
+          return;
+        }
+        return this.cloudController
+          .getServiceInstance(this.guid)
+          .then(instance => _.set(req, 'entity.name', instance.entity.name));
+      })
+      .then(() => req.manager.findDeploymentNameByInstanceId(req.params.instance_id))
+      .then(deploymentName => ({
+        instance_id: req.params.instance_id,
+        instance_name: req.entity.name,
+        deployment_name: deploymentName,
+        timeZone: req.body.timeZone
+      }))
+      .then((jobData) => ScheduleManager
+        .schedule(req.params.instance_id,
+          CONST.JOB.SERVICE_INSTANCE_UPDATE,
+          req.body.repeatInterval,
+          jobData,
+          req.user))
+      .then(body => res
+        .status(201)
```

Comment:
put status code in constants

Reply:
Same as [above](https://github.com/SAP/service-fabrik-broker/pull/59#discussion_r143665557)